src/components/SkyBrowser/InfoBoxSkybrowser.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { times } from 'lodash';
+import MaterialIcon from '../common/MaterialIcon/MaterialIcon';
+import Button from '../common/Input/Button/Button';
+import TooltipSkybrowser from './TooltipSkybrowser';
+import styles from './TooltipSkybrowser.scss';
+import esaSkyLogo from './ESASKY.png';
+
+class InfoBoxSkybrowser extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { showPopup: false, tooltipActive: false};
+    this.setRef = this.setRef.bind(this);
+    this.tooltipActive = this.tooltipActive.bind(this);
+    this.checkIfTooltipActive = this.checkIfTooltipActive.bind(this);
+    this.showPopup = this.showPopup.bind(this);
+    this.hidePopup = this.hidePopup.bind(this);
+    this.openImageUrl = this.openImageUrl.bind(this);
+    this.handleOutsideClick = this.handleOutsideClick.bind(this);
+    this.openEsaSky = this.openEsaSky.bind(this);
+  }
+
+
+  setRef(what) {
+    return (element) => {
+      this[what] = element;
+    };
+  }
+
+  get position() {
+    if (!this.wrapper) return { top: '0px', left: '0px' };
+    const { top, left, right, bottom } = this.wrapper.getBoundingClientRect();
+    return { top: `${top}`, left: `${right}`};
+  }
+
+  openImageUrl(imageUrl) {
+    const newWindow = window.open(imageUrl, '_blank', 'noopener,noreferrer');
+    if (newWindow) newWindow.opener = null;
+  }
+
+  handleOutsideClick(evt) {
+    if (this.wrapper && !this.wrapper.contains(evt.target)) {
+      this.hidePopup();
+    }
+  }
+
+  tooltipActive() {
+    this.setState({ tooltipActive: !this.state.tooltipActive });
+  }
+
+  checkIfTooltipActive() {
+    if(!this.state.tooltipActive) {
+        this.hidePopup();
+    }
+  }
+
+  showPopup() {
+    this.setState({ showPopup: !this.state.showPopup });
+  }
+
+  hidePopup() {
+    this.setState({ showPopup: false, tooltipActive: false});
+  }
+
+  openEsaSky(ra, dec, fov) {
+    let esaSkyUrl = "http://sky.esa.int/?target="+ra+"%"+dec+"&hips=DSS2+color&fov=" + fov + "&cooframe=J2000&sci=true&lang=en";
+    const newWindow = window.open(esaSkyUrl, "EsaSky");
+  }
+
+  render() {
+    const { icon, text, title, textUrl, ra, dec, fov} = this.props;
+    const { showPopup } = this.state;
+    const esaSkyButton =   <Button onClick={() => {this.openEsaSky(ra,dec,fov)}} className={styles.tooltipButton}  transparent small>
+        <img src={esaSkyLogo} alt="EsaSky" style={{width:'100%'}} />
+      </Button>;
+
+    return (
```

Comment:
Loths of indentation issues in this return

Not that we try to use 2 spaces for indentation throughout the UI